recipes/newmat/all/conanfile.py

```diff
@@ -0,0 +1,87 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, copy, get, rmdir, save, load
+from conan.tools.microsoft import is_msvc
+import os
+
+required_conan_version = ">=1.53.0"
+
+
+class NewmatConan(ConanFile):
+    name = "newmat"
+    package_type = "library"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://newmat.net"
+    description = "Manipulate a variety of types of matrices using standard matrix operations."
+    settings = "os", "compiler", "build_type", "arch"
+    topics = ("newmat", "matrix")
+    license = "LicenseRef-newmat"
+    settings = "os", "arch", "compiler", "build_type"
```

Comment:
Duplicated settings, unneeded topic

```suggestion
    name = "newmat"
    description = "Manipulate a variety of types of matrices using standard matrix operations."
    url = "https://github.com/conan-io/conan-center-index"
    homepage = "https://newmat.net"
    topics = ("matrix")
    license = "LicenseRef-newmat"
    package_type = "library"
    settings = "os", "arch", "compiler", "build_type"

```